faker/providers/misc/__init__.py

```diff
@@ -613,6 +614,28 @@ def create_json_structure(data_columns: Union[Dict, List]) -> dict:
         data = [create_json_structure(data_columns) for _ in range(num_rows)]
         return json.dumps(data, indent=indent, cls=cls)
 
+    def xml(self,
+        nb_elements: int = 10,
+        variable_nb_elements: bool = True,
+        value_types: Optional[TypesSpec] = None,
+        allowed_types: Optional[TypesSpec] = None,
+        ) -> str:
+        """
+        Returns a xml.
```

Comment:
Can you add that this features requires `xmltodict`? That way it will show up on our docs